### PR TITLE
Added unit tests for the forms module.

### PR DESCRIPTION
**Pull request by bolivierjr**
Refactored test_models.
Added a test for the short_id function in the core module.

### CODE REVIEW

**Comment by ChrisPappalardo** [comment]
thanks for the PR, i have some irl work stuff to do, will work on this over the weekend

**Comment by bolivierjr** [comment]
added more tests for the views to the PR. 92% coverage now.

**Comment by bolivierjr** [comment]
did some fixups

**Comment by bolivierjr** [comment]
Added the test for ChimeSiteView context. Coverage is now at 95%

**Comment by ChrisPappalardo** [comment]
Great, thanks Bruce.  Looking at it now...

File: tests/test_views.py [hunk 1]
```diff
@@ -98,3 +122,82 @@ def test_update_view_bad(self):
...
+        '''
+
+        self.client.login(username=f'testuser0', password=f'testuser0')
+        r = self.client.delete(f'/delete/{self.sites[0].id}/')
+        self.assertEqual(r.status_code, 302)
+
+        messages = list(get_messages(r.wsgi_request))
+        self.assertEqual(str(messages[0]), 'Successfully deleted CHIME app.')
+
+    def test_create_chimesite_view(self):
```
Comment (ChrisPappalardo) [inline tests/test_views.py]: I'm not sure what this is testing?  Groups is part of the Django internals.  Is this just seeing if we can instantiate ChimeSiteView()?

**Review by ChrisPappalardo — COMMENTED** [review]
Overall, everything looks good.  I made a few changes and pushed them, and have one question/comment to resolve.

File: tests/test_views.py (hunk 1 above)
Comment (bolivierjr) [reply tests/test_views.py]: It’s testing the get_context_data method that coverage was complaining about. Does the view not need to be instantiated to use that method? I can’t look at it at the moment

Comment (bolivierjr) [reply tests/test_views.py]: should have probably named the test function differently

Comment (ChrisPappalardo) [reply tests/test_views.py]: ok - im going to remove that one test pending a future one that tests the view more directly, and accept this PR.  going forward, you can use make coverage to test coverage (it was in the 80s after this PR iirc).

**Pull request merged**